DependencyInjection/DoctrineCouchDBExtension.php

```diff
@@ -63,8 +65,8 @@ public function getConfiguration(array $config, ContainerBuilder $container)
 
     private function clientLoad($config, ContainerBuilder $container)
     {
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-        $loader->load('client.xml');
+        $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('client.yml');
```

Comment:
Please keep XML as format. Symfony has no requirement for yaml and this is an unnecessary change.

Reply:
In hindsight it might be an overkill. I'll make those changes and submit it again.